packages/wallets/src/components/WalletCard/WalletCard.scss

```diff
@@ -1,8 +1,17 @@
+@import '../../components//SkeletonLoader/SkeletonLoader.scss';
```

Comment:
i think we should import from the component instead of directly on the css

Reply:
This is pure base css for the loader. All the other loaders are extending from this